### ISSUE TITLE
make: *** [html] Fehler 1

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#1283]
```
make livehtml
sphinx-build -b html -d sphinx/doctrees   . sphinx/html
Running Sphinx v1.1.3
loading pickled environment... done
building [html]: targets for 33 source files that are out of date
updating environment: 0 added, 0 changed, 1 removed
looking for now-outdated files... none found
preparing documents... done
writing output... [ 69&#37;] readme                                                                                                                                          
Exception occurred:
  File "/home/arinker/.virtualenvs/hgs.theme_leads/local/lib/python2.7/site-packages/sphinx/builders/html.py", line 407, in get_doc_context
    display_toc = (self.env.toc_num_entries[docname] > 1),
KeyError: 'readme'
The full traceback has been saved in /tmp/sphinx-err-h5qu6Q.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
Either send bugs to the mailing list at <http://groups.google.com/group/sphinx-dev/>,
or report them in the tracker at <https://bitbucket.org/birkenfeld/sphinx/issues/>. Thanks!
make: *** [html] Fehler 1
```

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/1283
- Originally reported by: Arno Rinker
- Originally created at: 2013-10-03T22:26:25.511

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2013-10-04 00:41:14+00:00_

I don't know what is `livehtml`, however I think it would be fixed when you run `make clean` once.

See also: #1267

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2014-01-19 16:33:28+00:00_

Closes #1283: Fix a bug in the detection of changed files that would try to access doctrees of deleted documents.

→ <<cset bc614bcd9356ed25e959b2f4fdf19551f0727c08>>
